lib/tensorflow.rb

```diff
@@ -1,6 +1,6 @@
 require 'tensorflow/core/framework/tensor'
 require 'tensorflow/core/framework/graph'
-require 'sciruby/Tensorflow'
+require 'sciruby/Internal'
```

Comment:
If it is possible with SWIG (I'm not sure) it is better to have here something like `require 'sciruby/tensorflow_internal'` or `require 'sciruby/tensorflow/internal'`

Because without further context, just `sciruby/Internal` looks really weird.